docs - update caching (#29401)
metabase/metabase: docs/questions/sharing/answers.md

![Save button](../images/save-button.png)

> A modal will appear, prompting you to give your question a name and description, and to pick which [collection](../../exploration-and-organization/collections.md) to save it in. Note that your administrator might have set things up so that you're only allowed to [save questions in certain collection](../../permissions/collections.md), but you can always save things in your Personal Collection. After saving your question, you'll be asked if you want to add it to a new or existing dashboard.

Reviewer: ```suggestion
A modal will appear, prompting you to give your question a name and description, and to pick which [collection](../../exploration-and-organization/collections.md) to save the question in. Note that your administrator might have set things up so that you're only allowed to [save questions in certain collection](../../permissions/collections.md), but you can always save items in your Personal Collection. After saving your question, you'll be asked if you want to add the question to a new or existing dashboard.
```

## Editing your question

Click into the question's title to edit the name of your question.

Open the **...** menu to:

> - [Verify](../../exploration-and-organization/exploration.md#verified-items) this question \*

Reviewer: ```suggestion
- [Verify](../../exploration-and-organization/exploration.md#verified-items) this question
```

- Add to [dashboard](../../dashboards/start.md)
- Move to another [collection](../../exploration-and-organization/collections.md)
- Turn into a [model](../../data-modeling/models.md)
- Duplicate
- [Archive](../../exploration-and-organization/history.md)

Click the **info** icon to:

- Add a description
- Edit the [cache duration](../../configuring-metabase/caching.md#caching-per-question)\*
- View [revision history](../../exploration-and-organization/history.md)

\* Available on [paid plans](https://www.metabase.com/pricing/).

## Bookmark a question

Click the **bookmark** icon to pin a question to your Metabase sidebar. See [Bookmarks](../../exploration-and-organization/exploration.md#bookmarks).

> ## Re-using a question

Reviewer: ```suggestion
## Building on existing saved questions
```
"Reusing" could imply copying, and that doesn't tell the whole story.
